swagger.yaml

```diff
@@ -0,0 +1,244 @@
+swagger: "2.0"
+info:
+  description: "discretionary-business-grants API Schema"
+  version: "0.0.1"
+  title: "discretionary-business-grants API"
+host: "discretionarybusinessgrants.hackney.gov.uk"
+basePath: "/api"
+tags:
+  - name: "urls"
+    description: "Secure URL for uploading files"
+  - name: "applications"
+    description: ""
+schemes:
+  - "https"
+paths:
+  /urls:
+    post:
+      tags:
+        - "urls"
+      summary: "Retrieve a new secure URL for file upload"
+      consumes:
+        - "application/json"
+      produces:
+        - "application/json"
+      parameters:
+        - name: "body"
+          in: "body"
+          required: true
+          schema:
+            $ref: "#/definitions/Url"
+      responses:
+        "500":
+          description: "Invalid input"
+        "200":
+          description: "Success"
+          schema:
+            type: "object"
+            items:
+              $ref: "#/definitions/UrlResponse"
+  /applications:
+    post:
+      tags:
+        - "applications"
+      summary: "Create a new business grant application"
+      consumes:
+        - "application/json"
+      produces:
+        - "application/json"
+      parameters:
+        - name: "body"
+          in: "body"
+          required: true
+          schema:
+            $ref: "#/definitions/Applications"
+      responses:
+        "500":
+          description: "Invalid input"
+        "200":
+          description: "Your application has been submitted"
+
+definitions:
+  Url:
+    type: "object"
+    properties:
+      id:
+        type: "integer"
+        format: "int64"
+  UrlResponse:
+    type: "object"
+    properties:
+      documentId:
+        type: "string"
+      fields:
+        type: "object"
+      url:
+        type: "string"
+
+  Applications:
+    type: "object"
+    properties:
+      clientGeneratedId:
+        type: "string"
```

Comment:
So this is the same id as referenced earlier in the definition of url. We should make the names consistent